meteor_packages/mats-common/imports/startup/api/matsMethods.js

```diff
@@ -875,7 +875,7 @@ const _saveResultData = function (result) {
         var sizeof = require('object-sizeof');
         var hash = require('object-hash');
         var key = hash(result.basis.plotParams);
-        var threshold = 1000000;
+        var threshold = 1200000;
```

Comment:
Small nit - but the comment for this function is now misleading since we're downsampling if the result's size is greater than 1.2Mb.